packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -240,6 +253,148 @@ const Authorization = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Get an OAuth Login URL for QRCode. Generate QR code based on the returned URL.
+   * @instance
+   * @memberof AuthorizationBrowserFirstParty
+   * @emits #qrcode-login
+   */
+  initQRCodeLogin() {
```

Comment:
since `_startQRCodePolling` is private and no point to get the QRCode details and then fail because there is a polling. 

Please add a check here for existing poll, `if (this.pollingRequest) {` and reject if there any.

Reply:
done